tests/unit_tests/steps/test_grab.py

```diff
@@ -42,41 +43,40 @@ def test_no_revision(self):
         source_root = Path('/workspace/source')
         source_url = '/www.example.com/bar'
         dst_label = 'bar'
-        grabber = GrabFcm(src=source_url, dst=dst_label)
+        grabber = FcmExport(src=source_url, dst=dst_label)
 
         mock_config = SimpleNamespace(source_root=source_root)
         with mock.patch('pathlib.Path.mkdir'):
-            with mock.patch('fab.steps.grab.fcm.run_command') as mock_run:
+            with mock.patch('fab.steps.grab.svn.run_command') as mock_run:
                 grabber.run(artefact_store={}, config=mock_config)
 
-        mock_run.assert_called_once_with(['fcm', 'export', '--force', source_url, str(source_root / dst_label)])
+        mock_run.assert_has_calls([
+            call(['fcm', 'help']),
```

Comment:
Is this left over debug code or is there a reason for it?

Reply:
No, it's a replacement for a little check that used to be done in the module upon import, which checked if the tool was available on the command line. After unifying svn and fcm, the check moved into the run method. It's not expensive but if we don't want to run this more than once we change it, if you prefer?